output/typescript/types.ts

```diff
@@ -753,7 +753,8 @@ export interface ICharFilter {
 
 export interface ITokenFilter {
   type: string
-  version: string
+  version?: string
+  stopwords?: Array<string>
```

Comment:
I dont think this should live on `ITokenFilter`. I'm reverting this which might turn indices.analyze red again @stevejgordon

Reply:
Ah okay. I'll take another run at it tomorrow.